app/static/js/slcNewProposal.js

```diff
@@ -57,7 +57,7 @@ $("#nextButton").on("click", function() {
 });
 
 $("#cancelButton").on("click", function() {
-  window.location.replace("/manageServiceLearning");
+        window.location.replace("/serviceLearning/courseManagement");
```

Comment:
Doesn't this depend on whether it is an admin creating the new proposal or a faculty member?